src/components/section/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react';
+import {
+  Box,
+  BoxProps,
+  chakra,
+  RecipeVariantProps,
+  useSlotRecipe,
+} from '@chakra-ui/react';
+
+import { sectionRecipe } from 'src/themes/shared/slotRecipes/section';
+
+import { Stack } from '../stack';
+
+type SectionVariantProps = RecipeVariantProps<typeof sectionRecipe>;
+
+export type SectionProps = SectionVariantProps & {
+  variant?: 'hero' | 'regular';
+  title: string;
+  text?: string;
+  image?: ReactNode;
+  maxImgW?: BoxProps['maxWidth'];
+};
+
+export const Section: FC<SectionProps> = ({ maxImgW = '2xl', ...props }) => {
+  // const recipe = useSlotRecipe({ key: 'section' });
+  const recipe = useSlotRecipe({ recipe: sectionRecipe });
+  const [recipeProps, componentProps] = recipe.splitVariantProps(props);
+  const styles = recipe(recipeProps);
+
+  const { title, text, image } = componentProps;
+
+  return (
+    <Stack direction={{ '2xs': 'column', md: 'row' }} gap="xl">
```

Comment:
Since we're creating a slot recipe, we could add a `root` slot and have all styles extracted to the recipe